templates/zerver/footer.html

```diff
@@ -44,48 +44,48 @@ <h3>{{ _("Product") }}</h3>
         <h3>{{ _("Solutions") }}</h3>
         <ul>
             <li>
-                <a href="/for/business/">{{ _("Business") }}</a>
+                <a href="https://zulip.com/for/business/">{{ _("Business") }}</a>
```

Comment:
We shouldn’t need to adjust the corporate footer’s links to corporate pages.

…Oh, I see. #22184 flipped the condition the wrong way around, so we’re incorrectly showing the self-hosted footer with corporate enabled, and the corporate footer with corporate disabled.

Reply:
Thanks for noticing. Addend another commit to fix the wrong condition. Also, removed the changes in footer.html in last commit.

Reply:
Merged this bit, thanks for catching this.